shared/utils/compare_remediations.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+if [ "$#" -lt 2 ];
+then
+	echo "Usage:"
+	echo -e "\t$0 <original repo> <updated repo> [meld]"
+	echo ""
+	echo -e "\tBoth repositories have to be already compiled! (make)"
+	echo -e "\tCompare <fix> elements from original DS with fixes in updated DSs"
+	exit 1
+fi
+
+originalRepo="$1/"
+updatedRepo="$2/"
+meld="$3"
+
+# Get list of remediations in pretty & sorted xml
+function extractRemediations() {
+	xsltproc $(dirname "$0")/getRemediations.xsl "$1"  | tee /tmp/res.xml | \
+		xmllint --c14n11 /dev/stdin | \
+		xmllint -format /dev/stdin | \
+		sed 's;^\s*#.*$;;g' | \
+		sed '/^\s*$/d'
+		# remove bash comments from output
+		# remove empty lines
+}
+
+function compareFile() {
+	local originalFile="$1"
+	toCompare=$(sed "s;^$originalRepo;$updatedRepo;" <<< "$originalFile")
+	echo "-----------------------------------------------------------------"
+	echo "$originalFile <=> $toCompare" 
+	echo "-----------------------------------------------------------------"
+
+	extractRemediations "$originalFile" > /tmp/original
+	extractRemediations "$toCompare"    > /tmp/new
+
+	if [ "$meld" == "meld" ];
+	then
+		diff /tmp/original /tmp/new -q || {
+			meld /tmp/original /tmp/new
```

Comment:
Does the ```meld``` binary called above come from [```meld``` rpm](https://admin.fedoraproject.org/pkgdb/package/rpms/meld/).

If that's the case I would suggest to add the check if ```meld``` rpm is installed on the system (from the above link it should be available in EPEL and recent Fedora, but needs to be installed). If ```meld``` isn't installed, a warning ```Please install meld package``` or something like that should be displayed (assuming right now it will show ```Command not found``` in this case).